<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nLineout</class>
 <widget class="QMainWindow" name="nLineout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>4</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="maxVal">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Higher value</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="minVal">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Lower value</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QwtPlot" name="the_plot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGetMinMax"/>
   <addaction name="actionAutoscale"/>
   <addaction name="actionMin"/>
   <addaction name="actionMax"/>
   <addaction name="actionToggleZoom"/>
   <addaction name="actionLockClick"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionToggleZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:icons//lockzoom</normaloff>
     <normalon>:/lockzoom2</normalon>:icons//lockzoom</iconset>
   </property>
   <property name="text">
    <string>toggleZoom</string>
   </property>
   <property name="toolTip">
    <string>Show visible part</string>
   </property>
  </action>
  <action name="actionAutoscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:icons//lockOpen</normaloff>
     <normalon>:/lockClose</normalon>:icons//lockOpen</iconset>
   </property>
   <property name="text">
    <string>autoscale</string>
   </property>
   <property name="toolTip">
    <string>Auto/FixedScale</string>
   </property>
  </action>
  <action name="actionGetMinMax">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:icons//center.png</normaloff>:icons//center.png</iconset>
   </property>
   <property name="text">
    <string>getMinMax</string>
   </property>
   <property name="toolTip">
    <string>Click mouse to set min and max</string>
   </property>
  </action>
  <action name="actionMin">
   <property name="text">
    <string>Min</string>
   </property>
  </action>
  <action name="actionMax">
   <property name="text">
    <string>Max</string>
   </property>
  </action>
  <action name="actionLockClick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/neutrino.qrc">
     <normaloff>:/icons/autoRefresh.png</normaloff>
     <normalon>:/icons/noAutoRefresh.png</normalon>:/icons/autoRefresh.png</iconset>
   </property>
   <property name="text">
    <string>lockClick</string>
   </property>
   <property name="toolTip">
    <string>update on click</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/neutrino.qrc"/>
 </resources>
 <connections/>
</ui>
